ar.com.softwareperonista.Rockarrolla.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "ar.com.softwareperonista.Rockarrolla",
```

Comment:
Just double-checking – this is an actual domain?

Reply:
yes, it is